packages/smart/deploy/008_deploy_master_chef.ts

```diff
@@ -0,0 +1,40 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { isHttpNetworkConfig, makeSigner } from "../tasks";
+import { Cash__factory, MasterChef__factory } from "../typechain";
+
+const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
+  const { deployments } = hre;
+
+  if (!isHttpNetworkConfig(hre.network.config)) {
+    return; // skip tests and internal deploy
+  }
+
+  const signer = await makeSigner(hre);
+  const deployer = await signer.getAddress();
+  const currentBlock = await hre.ethers.provider.getBlockNumber();
+
+  const wrappedMaticAddress =
+    hre.network.config.deployConfig?.externalAddresses?.wrappedMatic || (await deployments.get("WrappedMatic")).address;
+  const wrappedMatic = Cash__factory.connect(wrappedMaticAddress, signer);
+
+  const _rewardsPerBlock = 10;
+
+  const args: Parameters<MasterChef__factory["deploy"]> = [
+    wrappedMatic.address,
+    _rewardsPerBlock,
+    currentBlock,
+    currentBlock + 10000,
```

Comment:
Why 10000? Could use a constant with a nice name.

Reply:
We aren't doing bonus rewards so it can simply go away.